tool/get-language-repo.ts

```diff
@@ -32,7 +32,7 @@ export async function getLanguageRepo(
   // Workaround for https://github.com/shelljs/shelljs/issues/198
   // This file is a symlink which gets messed up by `shell.cp` (called from
   // `utils.link`) on Windows.
-  shell.rm('build/sass/spec/README.md');
+  if (process.platform === 'win32') shell.rm('build/sass/spec/README.md');
```

Comment:
Is there a reason why this is no longer necessary on Linux/Mac?

Reply:
It was only ever present as a workaround for a Windows issue in the first place (see the comment above).